source/docs/v3/tenants.blade.md

```diff
@@ -60,17 +59,29 @@ After the tenant is created, an event will be fired. This will result in things
 
 ## Custom columns {#custom-columns}
 
-Attributes of the tenant model which don't have their own column will be stored in the `data` JSON column.
+Attributes of the tenant model which don't have their column will be stored in the `data` JSON column. You can set these attributes as you'd normally set model attributes:
```

Comment:
- The first sentence doesn't need to be changed, add the `own` back
- updated second sentence
```suggestion
Attributes of the tenant model which don't have their own column will be stored in the `data` JSON column. You can set these attributes like you'd set normal model attributes:
```